rpcs3/util/asm.hpp

```diff
@@ -124,10 +128,12 @@ namespace utils
 			return;
 		}
 
-#ifdef _M_X64
-		return _mm_prefetch(static_cast<const char*>(ptr), 3);
+#ifdef _MSC_VER
+		return _mm_prefetch(static_cast<const char*>(ptr), _MM_HINT_T0);
+#elif _M_X64
+		return _mm_prefetch(ptr, _MM_HINT_T0);
 #else
-		return __builtin_prefetch(ptr, 0, 3);
+		return __builtin_prefetch(ptr, 0, _MM_HINT_T0);
```

Comment:
why are these changed from 3 to _MM_HINT_T0, which is 1 ?

Reply:
GCC and Clang define `_MM_HINT_T0 = 3` in `xmmintrin.h`, so there's no change here.

Reply:
Reverted too.